### DISCUSSION TITLE
Sending messages with a new line are not sent (Twilio)

### DISCUSSION THREAD

**Discussion by yairShiloach** [chatwoot/chatwoot#2626] [General]
We have a problem with sending templates,
As is well known, a message to a phone number can only be sent via WhatsApp if a template format is approved.

When I confirm a message that has a new line, and try to send it through the app, the message is not sent claiming it does not match the format.
chatwoot does some character conversion, and deletes the lines (using api)
How can I send a message using an api to be sent to a customer?

**Comment by pranavrajs** [comment]
@yairShiloach Would you be able to share the template you are using?

**Reply by yairShiloach** [reply]
sure!
when I try to send the template from postmark (with Twilio api)
its look like:
Hello {{1}}\nWe wanted to update you that we closed the support ticket: {{2}}

the template:
Hello {{1}}
We wanted to update you that we closed the support card: {{2}}

**Reply by pranavrajs** [reply]
@yairShiloach This looks like a bug, can you please try using the API https://www.chatwoot.com/developers/api#operation/create-a-new-message-in-a-conversation and see if the error is still reproducible?
